test/config.yaml

```diff
@@ -1,6 +1,7 @@
 --- 
 dataDir: /tmp/microshift/data
 auditLogDir: /tmp/microshift/logs
+lvmdConfigFile:
```

Comment:
I think we want to remove this line, right?

Reply:
Ah, yes.  It's ignored in the tests since the field doesn't exist on the go side. Will fix.